protocol-units/settlement/mcr/contracts/script/DeployMoveTokenMultisig.s.sol

```diff
@@ -0,0 +1,108 @@
+pragma solidity ^0.8.13;
+
+import "forge-std/Script.sol";
+import {MOVEToken} from "../src/token/MOVEToken.sol";
+import {ProxyAdmin} from "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+import {TransparentUpgradeableProxy} from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import {SafeProxyFactory} from "@safe-smart-account/contracts/proxies/SafeProxyFactory.sol";
+import { SafeProxy } from "@safe-smart-account/contracts/proxies/SafeProxy.sol";
+import {Safe} from "@safe-smart-account/contracts/Safe.sol";
+import {CreateCall} from "@safe-smart-account/contracts/libraries/CreateCall.sol";
+import {TimelockController} from "@openzeppelin/contracts/governance/TimelockController.sol";
+import {Enum} from "@safe-smart-account/contracts/common/Enum.sol";
+
+contract DeployMoveTokenMultisig is Script {
+    TransparentUpgradeableProxy public moveProxy;
+    ProxyAdmin public admin;
+    string public moveSignature = "initialize(address)";
+    string public safeSetupSignature = "setup(address[],uint256,address,bytes,address,address,uint256,address)";
+    SafeProxyFactory public safeProxyFactory;
+    address public zero = address(0x0);
+    address public safeSingleton = 0x29fcB43b46531BcA003ddC8FCB67FFE91900C762;
+    CreateCall public createCall = CreateCall(0x7cbB62EaA69F79e6873cD1ecB2392971036cFAa4);
+    address payable public safeAddress;
+    Safe public safe;
+    uint256 public threshold = 2;
+    TimelockController public timelock;
+
+    function generateSignatures(uint256 privKey, bytes32 digest) internal returns (bytes memory signatures) {
+        (uint8 v1, bytes32 r1, bytes32 s1) = vm.sign(privKey, digest);
+        (uint8 v2, bytes32 r2, bytes32 s2) = vm.sign(2, digest);
+        (uint8 v3, bytes32 r3, bytes32 s3) = vm.sign(3, digest);
+
+        console.log("v1: ", v1);
+        console.log("v2: ", v2);
+        console.log("v3: ", v3);
+        signatures = abi.encodePacked(r1, s1, v1, r2, s2, v2);
+    }
+
+    function run() external {
```

Comment:
This needs to be chunked--either with sections clearly delineated by comments or with helper methods.

Reply:
Not sure if I understood this one. What needs to be chunked?

Reply:
dropped this bellow as it's just utility for this specific contract. No need to move it elsewhere.

Reply:
This run method is too verbose. It should be broken into smaller helper methods.

Reply:
already cleared.